DistTasks.yml

```
@@ -19,7 +19,7 @@ version: "3"
 
 vars:
   CONTAINER: "docker.elastic.co/beats-dev/golang-crossbuild"
-  GO_VERSION: "1.16.4"
+  GO_VERSION: "1.17.3"
```

```suggestion
  GO_VERSION: "1.17.8"
```
This is the current latest available:
https://github.com/elastic/golang-crossbuild/releases/tag/1.17.8

Unless there is a specific reason for using 1.17.3, I suggest we get all the way up to date while we are at it.